CHANGELOG.md

```diff
@@ -7,11 +7,16 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## Unreleased
 
+<!-- List New Changes Here -->
+
+## [0.7.0] - 2017-01-12
```

Comment:
Ok, will do. I was anticipating your change in your PR for 0.7.0, which would result in a merge conflict here.